src/main/java/menion/android/whereyougo/utils/NotificationService.java

```diff
@@ -0,0 +1,123 @@
+package menion.android.whereyougo.utils;
+
+import android.app.Notification;
+import android.app.NotificationChannel;
+import android.app.PendingIntent;
+import android.app.Service;
+import android.app.NotificationManager;
+import android.content.Context;
+import android.content.Intent;
+import android.os.Build;
+import android.os.IBinder;
+import android.support.v4.app.NotificationCompat;
+
+public class NotificationService extends Service {
+    private static final int notification_id = 10;
+    private static final String NOTIFICATION_CHANNEL_ID = "menion.android.whereyougo.utils.NotificationService";
+    private boolean foreground = false;
+    private boolean runing = false;
+    private NotificationManager mNM;
+    private String contentTitel;
+
+    public static final String TITEL = "ContentTitel";
```

Comment:
Is TITEL visible to users?

Reply:
No, the TITEL constant is not visible to the user. It is ussed for the internal tagging of data.